superset/charts/schemas.py

```diff
@@ -284,7 +285,9 @@ class ChartPutSchema(Schema):
     )
     is_managed_externally = fields.Boolean(allow_none=True, dump_default=False)
     external_url = fields.String(allow_none=True)
-    tags = fields.Nested(TagSchema, many=True)
```

Comment:
Where are we removing the `TagSchema` reference here?

Reply:
that's a good catch -- I scanned for un-used imports but didn't realize that `TagSchema` was actually defined in this same file. I'll remove its definition in a follow up commit.

Reply:
I'll hold on committing this change ☝️ until I get your feedback on the other comment.

Reply:
So we should prolly centralize `TagSchema` somewhere it could be used in all the resource update schemas